boards/shields/esp_8266/boards/numaker_pfm_m467.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 Gerson Fernando Budke <nandojve@gmail.com>
```

Comment:
You can add you copyright here. No need to keep mine in this case.

Reply:
fixed copyright